src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -946,35 +946,53 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: SlickDataSo
 
         val submissionId: String = UUID.randomUUID().toString
         val workflowOptionsFuture = buildWorkflowOptions(workspaceContext, submissionId)
-        val submittedWorkflowsFuture = workflowOptionsFuture.flatMap(workflowOptions => Future.sequence(successes map { entityInputs =>
+
```

Comment:
note that there are new 2 forms of failure submitting to cromwell
1. the http call fails with some error status
2. individual workflows within the batch fail but overall the http request is a success